deps/chakrashim/include/v8-debug.h

```diff
@@ -91,7 +91,7 @@ class V8_EXPORT Debug {
   static Local<Context> GetDebugContext(Isolate* isolate);
 
   static void EnableDebug();
-  static void EnableInspector();
+  static void EnableInspector(bool enableReplayDebug = false);
```

Comment:
If it was present, it no longer is.  Looking at the usage, I suspect we added it as a workaround.